frontend/src/metabase/components/Modal/Modal.jsx

```diff
@@ -30,6 +30,7 @@ export class WindowModal extends Component {
     isOpen: PropTypes.bool,
     enableMouseEvents: PropTypes.bool,
     enableTransition: PropTypes.bool,
+    noOnClickOutsideWrapper: PropTypes.bool,
```

Comment:
Could we call it `closeOnClickOutside` or something similar, which would describe the behavior? We could make it `true` by default via `defaultProps`, so we won't have to add a property starting with `no`.

Reply:
Yep, I was the feeling the same way, I just wanted to keep it consistent with the `Popover` since it has the same prop. I'll change it though, it makes more sense